core/chains/evm/config/chain_specific_config.go

```diff
@@ -285,26 +285,17 @@ func setChainSpecificConfigDefaultSets() {
 	arbitrumMainnet.blockEmissionIdleWarningThreshold = 0
 	arbitrumMainnet.nodeDeadAfterNoNewHeadersThreshold = 0 // Arbitrum only emits blocks when a new tx is received, so this method of liveness detection is not useful
 	arbitrumMainnet.chainType = config.ChainArbitrum
-	arbitrumMainnet.gasBumpThreshold = 0 // Disable gas bumping on arbitrum
-	arbitrumMainnet.gasLimitDefault = 7000000
-	arbitrumMainnet.gasLimitTransfer = 800000            // estimating gas returns 695,344 so 800,000 should be safe with some buffer
-	arbitrumMainnet.gasPriceDefault = *assets.GWei(1000) // Arbitrum uses something like a Vickrey auction model where gas price represents a "max bid". In practice we usually pay much less
-	arbitrumMainnet.maxGasPriceWei = *assets.GWei(1000)  // Fix the gas price
-	arbitrumMainnet.minGasPriceWei = *assets.GWei(1000)  // Fix the gas price
-	arbitrumMainnet.gasEstimatorMode = "FixedPrice"
+	arbitrumMainnet.gasBumpThreshold = 0               // Disable gas bumping on arbitrum
+	arbitrumMainnet.gasPriceDefault = *assets.Wei(1e8) // 0.1 gwei
+	arbitrumMainnet.maxGasPriceWei = *assets.GWei(1000)
+	arbitrumMainnet.minGasPriceWei = *assets.GWei(0)
```

Comment:
Arbitrum mentioned that the min value for baseFee (which is the same as gas price for Arbitrum) is 0.1 gwei

Reply:
Yes but it could change, and there is no need to over-constrain.